RZViewActions/UIView+RZViewActions.m

```diff
@@ -90,9 +97,16 @@ + (instancetype)action:(RZViewActionBlock)action withDuration:(NSTimeInterval)du
 
 + (instancetype)action:(RZViewActionBlock)action withOptions:(UIViewAnimationOptions)options duration:(NSTimeInterval)duration
 {
-    NSParameterAssert(action);
+    return [[RZViewAction alloc] initWithBlock:action options:options duration:duration];
+}
+
++ (instancetype)springAction:(RZViewActionBlock)action withDamping:(CGFloat)dampingRatio initialVelocity:(CGFloat)velocity options:(UIViewAnimationOptions)options duration:(NSTimeInterval)duration
```

Comment:
Shouldn’t this return `RZViewSpringAction`? And that raises the question of whether this method should perhaps be on `RZViewSpringAction` instead of `RZViewAction`?